playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -186,7 +186,9 @@
         SHIPYARD_IMAGE: "{{ shipyard_image }}"
         OS_PASSWORD: "{{ lookup('password', secrets_location + '/ucp_shipyard_keystone_password ' + password_opts) }}"
       register: shipyard_desc_action
-      failed_when: "'failed' in shipyard_desc_action.stdout"
+      failed_when: >
+          "'failed' in shipyard_desc_action.stdout" or
+          "'Error' in·shipyard_desc_action.stdout"
```

Comment:
do we want to fail out when this happens? because sometimes is just a retryable error like the temporary 503 that we get..

For that we should modify the until I guess

Reply:
I think you are right, but it's gonna look ugly to check for error there. Fixing.

Reply:
The until is just to tell you when to stop, failure or not.
That until raises my eyebrows, there is no test to find in the stdout that things were fine? Just the absence of "running" or "Processing"? That's weird.